formats.js

```diff
@@ -5,7 +5,7 @@
  *
  * THIS IS A BACKUP FILE.
  */
- exports.BattleFormats = {
+exports.Formats = [
```

Comment:
It's a backup, it tries to keep the same code as the server. I mean, it's not plug and play code, but it should follow the same style.

The other changes following gulpfile are good, thanks!